src/translate/function.rs

```diff
@@ -81,6 +81,30 @@ pub fn translate_function(sql_function: &SqlFunction) -> Result<Expr> {
 
             Ok(Expr::Function(Box::new(Function::Right { expr, size })))
         }
+        "CEIL" => {
+            check_len(name, args.len(), 1)?;
+
+            translate_expr(args[0])
+                .map(Function::Ceil)
+                .map(Box::new)
+                .map(Expr::Function)
+        }
+        "ROUND" => {
+            check_len(name, args.len(), 1)?;
+
+            translate_expr(args[0])
+                .map(Function::Round)
+                .map(Box::new)
+                .map(Expr::Function)
+        }
+        "FLOOR" => {
+            check_len(name, args.len(), 1)?;
+
+            translate_expr(args[0])
+                .map(Function::Floor)
+                .map(Box::new)
+                .map(Expr::Function)
+        }
```

Comment:
I made a macro called func_with_one_arg.
You can add it comfortably.
Rebase and use the master.
Below is an example of usage.

```rust
func_with_one_arg!(Function::Ceil)
```

Reply:
Can be used when it's a function and only has one argument

Reply:
Oh thank you, i'll use that macro that can remove redundant code